Cargo.toml

```diff
@@ -72,6 +72,8 @@ ibc-core-host = { git = "https://github.com/mina86/ibc-rs", rev = "2b50319f3460d
 ibc-core-host-types = { git = "https://github.com/mina86/ibc-rs", rev = "2b50319f3460d03e8781e5a834c4b56bdcfdc5bc", default-features = false }
 ibc-primitives = { git = "https://github.com/mina86/ibc-rs", rev = "2b50319f3460d03e8781e5a834c4b56bdcfdc5bc", default-features = false }
 ibc-testkit = { git = "https://github.com/mina86/ibc-rs", rev = "2b50319f3460d03e8781e5a834c4b56bdcfdc5bc", default-features = false }
+solana-sdk = { git = "https://github.com/ComposableFi/mantis-solana", rev = "d92c52575aaf430ca1f06bc2589f26b0fedde7bf" }
```

Comment:
I’m a bit sceptical of using solana-sdk. The light client runs as a smart contract on Cosmos chain. Can we make this work with solana-program?

Reply:
It's just to reuse some definitions from there like VersionedTransaction. It supports no-std, so I guess we there won't be any issues deploying it as a cosmwasm contract